easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCCcore-5.3.0.eb

```diff
@@ -25,6 +28,12 @@ configopts = [
     '--with-shared --enable-overwrite --enable-ext-colors --enable-widec --includedir=%(installdir)s/include/ncursesw/'
 ]
 
+# Symlink libtinfo to libncurses
+postinstallcmds = [
+    "ln -s %(installdir)s/lib/libncurses.so %(installdir)s/lib/libtinfo.so",
+    "ln -s %(installdir)s/lib/libncurses.a %(installdir)s/lib/libtinfo.a"
```

Comment:
I think there's a good reason we only do this when building `ncurses` with `dummy`, but I don't recall the details.

Maybe @ocaisa does?

Reply:
https://github.com/easybuilders/easybuild-easyconfigs/pull/5067 has the context but I really don't remember the details.